backend/src/core/environment/environment.ts

```diff
@@ -16,13 +16,15 @@ export class Environment implements IEnvironment {
   public readonly port: number;
   public readonly telnetHost: string;
   public readonly telnetPort: number;
+  public readonly telnetTLS: boolean;
```

Comment:
Kann man man da echt einfach nur ein boolean reinhauen und plötzlich hat man TLS? Wird das alles "under the hood" ausgehandelt? Wenn ja, dann lass uns das andere, zertfikat basierte TLS raushauen. Wenn wir dann die App ausliefern wollen mit SSL, müssen wir das sowieso anders lösen. Die TLS Einstellung wäre dann wenn dann nur für den Socket notwendig, wenn überhaupt.

Reply:
ich habe nicht verstanden, WARUM Du bei einem Telnet CLIENT ein TSL ZErtifkat haben willst.

Das TLS wird komplett mit gültigem Zertifkat vom Server auf dem richtigen Port zur Verfügung gestellt siehe auch "hilfe ssl" im mud. Ob wir später ein CLient Zertifikat brauchen können, hilft vom ORT des backends ab. Solange das Backend unter der Kontrolle der Serveradmins des Muds ist, brauchen wir das nicht.

Option wäre, dass Du ein Client Zertifikat brauchst, um ein Backend zb bei Azure betreiben zu dürfen, müssten wir aber mit den admins diskutieren.